### PR TITLE
RD-4292 CLI for listing operations/graphs

### PR DESCRIPTION
**Pull request by tehasdf**
Give the user a way to look at their operations and graphs. This is
going to give them more insight into their executions, and eg. what
operations caused an execution to fail.

### CODE REVIEW

File: cloudify_cli/commands/executions.py [hunk 1]
```diff
@@ -469,6 +469,65 @@ def delete(logger, client, tenant_name, to_datetime, before, keep_last,
...
+
+@graphs.command('list')
+@cfy.options.common_options
+@cfy.argument('execution-id', required=True)
+@click.option('--name', help='List graphs with this name')
+@cfy.pass_logger
+@cfy.pass_client()
+def graphs_list(execution_id, name, client, logger):
+    """List tasks-graphs for an execution"""
+    tgs = client.tasks_graphs.list(execution_id=execution_id, name=name)
```
[the remark this answers is not in the document]
Comment (geokala) [reply cloudify_cli/commands/executions.py:486]: But the name None, as provided by the CLI, is not None, it's "None".

[the remark this answers is not in the document]
Comment (tehasdf) [reply cloudify_cli/commands/executions.py:486]: nope!

[the remark this answers is not in the document]
Comment (tehasdf) [reply cloudify_cli/commands/executions.py:486]: see eg. `test_list_name` where it does assert that `not name`

[the remark this answers is not in the document]
Comment (geokala) [reply cloudify_cli/commands/executions.py:486]: Oh, I missed that it's not required, OK, nice.

**Review by geokala — APPROVED** [review]

**Pull request merged**